options.js

```diff
@@ -66,3 +66,20 @@ exports.getDefaults = function getDefaults(features) {
 
   return options;
 };
+
+exports.getMinifier = function getMinifier() {
```

Comment:
I realize we're inside `options.js`, but my only ask might be a different name for this as it's not returning an actual minifier, but rather the default options.  Maybe `getMinifierDefaults`?